content/en/cosign/overview.md

```diff
@@ -98,18 +148,9 @@ Enter password for private key:
 Pushing signature to: index.docker.io/user/demo:sha256-87ef60f558bad79be4def8.sig
 ```
 
-### Other Formats
+## Other Formats
 
 Cosign is useful not only for blobs, containers, and container-related artifacts; it can also be used for other file types.
 
-To learn how to sign SBOMs, WASM modules, Tekton bundles and more, review [Signing Other Types](/cosign/other_types/). For more information about blobs, review [Working with Blobs](/cosign/working_with_blobs/).
-
-### SCM Integration
-
-Cosign integrates natively with source code management (SCM) systems like GitHub and GitLab. You can use the official [GitHub Actions Cosign installer](https://github.com/marketplace/actions/cosign-installer) or use cosign to generate and work safely with [SCM secrets](/cosign/git_support/) with native API integration.
-
-### Attestations
-
-In addition to signatures, Cosign can be used with [In-Toto Attestations](https://github.com/in-toto/attestation).
+To learn how to sign SBOMs, WASM modules, Tekton bundles and more, review [Signing Other Types](/cosign/other_types/). For more information about blobs, review [Signing Blobs](/cosign/signing_with_blobs/). For containers, see [Signing Containers](/cosign/signing_with_containers/).
 
-Attestations provide an additional semantic-layer on top of plain cryptographic signatures that can be used in policy systems.
```

Comment:
Can you add the line: 

Learn more in the [Attestations](https://docs.sigstore.dev/cosign/attestation) documentation.

Reply:
added